photo/tests/test_mutations/test_collection.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+from django.test import TestCase
+
+from photo.schema import schema
+from photo.tests.factories import CollectionFactory, PictureFactory, UserFactory
+from .mutation_file import (
+    collection_add_picture_mutation,
+    collection_creation_mutation,
+    collection_update_mutation,
+)
+
+
+class CollectionTest(TestCase):
+    def setUp(self):
+        self.newUser = UserFactory(user_profile_picture=True)
+        self.newPictures = PictureFactory.create_batch(10, user=self.newUser)
+
+    @pytest.mark.asyncio
+    async def test_create_one(self):
+        mutation = collection_creation_mutation
+        newUser = self.newUser
+
+        newCollection = {
+            "user": str(newUser.id),
+            "name": "Best collection",
+            "pictures": [picture.id for picture in self.newPictures],
+        }
+
+        result = await schema.execute(
+            mutation,
+            variable_values={"collection": newCollection},
+        )
+
+        self.assertEqual(result.errors, None)
+        self.assertEqual(
+            result.data["create_collection"]["user"]["id"], str(newUser.id)
+        )
+        self.assertEqual(
+            len(result.data["create_collection"]["pictures"]), len(self.newPictures)
+        )
+        for picture in result.data["create_collection"]["pictures"]:
+            self.assertTrue(picture["id"] in newCollection["pictures"])
+        self.assertEqual(
+            result.data["create_collection"]["name"], newCollection["name"]
+        )
+
+    @pytest.mark.asyncio
+    async def test_create_fail(self):
+        mutation = collection_creation_mutation
+        newUser = self.newUser
+
+        newCollection = {
+            "user": str(newUser.id),
+            "name": "Best collection",
+            "pictures": [picture.id for picture in self.newPictures],
+        }
+
+        newCollection2 = {
```

Comment:
this is not necessary, the schema can be called using the previously created collection